lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -1933,7 +2006,7 @@ bool EarlyFunctionAnalysis::runOnModule(Module &M) {
 
   // Create an over-approximated call graph of the program. A queue of all the
   // function entrypoints is maintained.
-  UniquedQueue<BasicBlockNode *> EntrypointsQueue;
+  BasicBlockQueue EntrypointsQueue;
```

Comment:
I think we should push the ownership of `EntrypointsQueue` into `Analyzer`. The queue is a concept that only makes sense *within* the analyzer.

So, either push the initialization of the queue in the analyzer too, or make the analyzer accept a `set` instead of a queue.